alpha/template/composite/builder.go

```diff
@@ -78,19 +82,21 @@ func (bb *BasicBuilder) Build(dir string, td TemplateDefinition) error {
 		return fmt.Errorf("basic template configuration is invalid: %s", strings.Join(validationErrs, ","))
 	}
 
-	// build the container command
-	containerCmd := exec.Command(bb.builderCfg.ContainerCfg.ContainerTool,
-		"run",
-		"--rm",
-		"-v",
-		fmt.Sprintf("%s:%s:Z", bb.builderCfg.CurrentDirectory, bb.builderCfg.ContainerCfg.WorkingDir),
-		bb.builderCfg.ContainerCfg.BaseImage,
-		"alpha",
-		"render-template",
-		"basic",
-		path.Join(bb.builderCfg.ContainerCfg.WorkingDir, basicConfig.Input))
-
-	return build(containerCmd, path.Join(dir, basicConfig.Output), bb.builderCfg.OutputType)
+	b := basictemplate.Template{Registry: reg}
+	reader, err := os.Open(basicConfig.Input)
```

Comment:
I think we may need to also include the current directory like we did when mounting volumes previously:
```suggestion
	reader, err := os.Open(path.Join(bb.builderCfg.CurrentDirectory, basicConfig.Input))
```